File: imagej-1/macros/Examples/Python/Sphere.py
t0 = System.currentTimeMillis()
size = 512
ip = FloatProcessor(size,size)
for y in range(size):
   IJ.showProgress(y,size-1)
   for x in range(size):
       dx=x-size/2; dy=y-size/2
       d = Math.sqrt(dx*dx+dy*dy)
       ip.setf(x,y,-d)
time = str((System.currentTimeMillis()-t0)/1000.0)+" seconds"
ImagePlus(time,ip).show()
IJ.run("Red/Green");

File: imagej-1/macros/Examples/JavaScript/Sphere.js
  t0 = System.currentTimeMillis();
  size = 512;
  ip = new FloatProcessor(size,size);
  for (y=0; y<size; y++) {
     IJ.showProgress(y,size-1);
     for (x=0; x<size; x++) {
        dx=x-size/2; dy=y-size/2;
        d = Math.sqrt(dx*dx+dy*dy);
        ip.setf(x,y,-d);
     }
  }
  time = (System.currentTimeMillis()-t0)/1000+" seconds";
  img = new ImagePlus(time,ip);
  IJ.run(img,"Red/Green","");
  img.show();
